src/Backend/DependencyInjection/BackendExtension.php

```diff
@@ -30,13 +30,20 @@ public function prepend(ContainerBuilder $container)
                 continue;
             }
 
-            // @TODO Check for YAML/XML files and set the type accordingly
+            /*
+             * Find and load entities in the backend folder
+             * We do this by looping all installed modules and looking for an Entity map If the Entity map is found,
```

Comment:
I'd add a point after `map` and a break before `If`.

I think `map` is also only correct in Dutch, nog in English. It could also be correct in Dunglish.